src/components/Text/textChangelogData.ts

```diff
@@ -9,6 +9,13 @@
 import { ChangelogData } from "../../utils/ComponentChangelogTable";
 
 export const changelogData: ChangelogData[] = [
+  {
+    date: "Prerelease",
+    version: "Prerelease",
+    type: "Update",
+    affects: ["Styles"],
+    notes: ["Chakra 2.8 update."],
+  },
   {
     date: "Prerelease",
     version: "Prerelease",
```

Comment:
Is this showing `"Prerelease"` because this branch needs to be updated with the latest from `development`?

Reply:
Yes. The main feature branch has not been updated just yet.